### PR TITLE
feat(ui): [KDL6-130] remove repository type 

### PR DESCRIPTION
**Pull request by joancarballo**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] I have updated the documentation accordingly.
- [ ] The commit message follows our guidelines: https://github.com/konstellation-io/kdl-server/blob/main/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: app/ui/src/Graphql/client/hooks/useNewProject.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import {
-  ExternalRepositoryValues,
-  ExternalRepositoryErrors,
-  InternalRepositoryValues,
-  InternalRepositoryErrors,
+  // ExternalRepositoryValues,
```
Comment (igzjaviergil) [inline app/ui/src/Graphql/client/hooks/useNewProject.ts:2]: letś remove evertyhing and avoid leaving comments in code

File: app/ui/src/Mocks/entities/project.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { memberMe, member1, member2 } from './member';
-import { RepositoryType } from '../../Graphql/types/globalTypes';
+// import { RepositoryType } from '../../Graphql/types/globalTypes';
```
Comment (igzjaviergil) [inline app/ui/src/Mocks/entities/project.ts:2]: same here

File: app/ui/src/Pages/NewProject/NewProject.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import DefaultPage from 'Components/Layout/Page/DefaultPage/DefaultPage';
 import Information from './pages/Information/Information';
 import ROUTE from 'Constants/routes';
 import RepositoryDetails from './pages/RepositoryDetails/RepositoryDetails';
-import { RepositoryType } from 'Graphql/types/globalTypes';
+// import { RepositoryType } from 'Graphql/types/globalTypes';
```
Comment (igzjaviergil) [inline app/ui/src/Pages/NewProject/NewProject.tsx:9]: ditto

File: app/ui/src/Pages/Project/pages/Overview/Overview.tsx [hunk 4]
```diff
@@ -7,7 +7,7 @@ import RepositoryTypeComponent, {
 import DescriptionScore from 'Components/DescriptionScore/DescriptionScore';
 import { GetProjects_projects } from 'Graphql/queries/types/GetProjects';
 import * as React from 'react';
-import { RepositoryType } from 'Graphql/types/globalTypes';
+// import { RepositoryType } from 'Graphql/types/globalTypes';
```
Comment (igzjaviergil) [inline app/ui/src/Pages/Project/pages/Overview/Overview.tsx:10]: ditto

File: app/ui/src/Pages/NewProject/pages/Repository/Repository.tsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import RepositoryTypeComponent, { LOCATION } from './components/RepositoryTypeComponent/RepositoryTypeComponent';
 import * as React from 'react';
 import RepositoryOption from './components/RepositoryOption/RepositoryOption';
-import { RepositoryType } from 'Graphql/types/globalTypes';
+// import { RepositoryType } from 'Graphql/types/globalTypes';
```
Comment (igzjaviergil) [inline app/ui/src/Pages/NewProject/pages/Repository/Repository.tsx:4]: ditto

File: app/ui/src/Pages/Projects/components/Project/Project.tsx [hunk 6]
```diff
@@ -92,7 +92,7 @@ const LowerBg: FC<BaseProps> = ({ project }) => (
 
 const Band: FC<BaseProps> = ({ project }) => (
   <div className={styles.band}>
-    <div className={styles.label}>{capitalize(project.repository?.type)}</div>
+    {/* <div className={styles.label}>{capitalize(project.repository?.type)}</div> */}
```
Comment (igzjaviergil) [inline app/ui/src/Pages/Projects/components/Project/Project.tsx:95]: ditto

**Review by igzjaviergil — CHANGES_REQUESTED** [review]

File: app/ui/src/Graphql/client/hooks/useNewProject.ts (hunk 1 above)
Comment (joancarballo) [reply app/ui/src/Graphql/client/hooks/useNewProject.ts]: Deleted this and related.

File: app/ui/src/Mocks/entities/project.ts (hunk 2 above)
Comment (joancarballo) [reply app/ui/src/Mocks/entities/project.ts]: Deleted this and related.

File: app/ui/src/Pages/Project/pages/Overview/Overview.tsx (hunk 4 above)
Comment (joancarballo) [reply app/ui/src/Pages/Project/pages/Overview/Overview.tsx]: Deleted this and related.

File: app/ui/src/Pages/NewProject/NewProject.tsx (hunk 3 above)
Comment (joancarballo) [reply app/ui/src/Pages/NewProject/NewProject.tsx]: Deleted this and related.

File: app/ui/src/Pages/NewProject/pages/Repository/Repository.tsx (hunk 5 above)
Comment (joancarballo) [reply app/ui/src/Pages/NewProject/pages/Repository/Repository.tsx]: Deleted this and related.

File: app/ui/src/Pages/Projects/components/Project/Project.tsx (hunk 6 above)
Comment (joancarballo) [reply app/ui/src/Pages/Projects/components/Project/Project.tsx:95]: Deleted this and related.

**Review by igzjaviergil — APPROVED** [review]

**Pull request merged**